Modules/System/Barcode/src/Barcode Provider 2D/BarcodeEncodeSettings2D.Table.al

```diff
@@ -0,0 +1,54 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Common setting used when encoding @D barcodes.
+/// </summary>
+table 9204 "Barcode Encode Settings 2D"
```

Comment:
Are these settings indeed for 2D barcodes, or are they for your QR code implementation? I they are for QR code only, I'm not sure if I'm a fan...

Reply:
...but then again, we did the same with our 1D barcodes as well. And I didn't particularly liked it there either ;-) But we should probably be consistent and do it here as well. So I guess I'm ok with this.

Reply:
I hate this solution. But as you mentioned it is the same way as the 1D implementation.

Reply:
Then we're in agreement. We close our eyes and push forward ;-)